src/main/java/fr/lunatech/timekeeper/models/time/TimeEntry.java

```diff
@@ -40,6 +41,19 @@
     public LocalDateTime startDateTime;
 
     @NotNull
-    public Integer numberOfHours;
-
+    public LocalDateTime endDateTime;
+
+    public Long getRoundedNumberOfHours() {
+        if(startDateTime == null ) {
```

Comment:
all there could be combined to one as all returns 0L
`if(startDateTime == null || endDateTime==null || endDateTime.isBefore(startDateTime)){
            return 0L;
        }`

Reply:
ok i will change it